src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/** Storage is a class that help store data in the local file and also read data from local file,
+ * @author CShuwen
+ * @version 1.0
+ * @since 0.0
+ */
+public class Storage {
+    private final String data_address;
+    public Storage(String file_name) {
```

Comment:
According to the coding standard about [naming variables](https://se-education.org/guides/conventions/java/intermediate.html#naming), variable names should be camelCase instead of snake_case.